python/triton/tools/aot.py

```diff
@@ -50,6 +50,13 @@
         print(module)
         exit(0)
 
+    # llvm-ir -> amdgcn
```

Comment:
please move this part to the end of the file so that it's in its own logical section instead of wedging it here in between the ptx argument checks and the llvm-ir -> ptx generation code.